bimmer_connected/vehicle_status.py

```diff
@@ -162,7 +162,7 @@ def is_vehicle_tracking_enabled(self) -> bool:
 
         The server return "OK" if tracking is enabled and "DRIVER_DISABLED" if it is disabled in the vehicle.
         """
-        return self._state.attributes[SERVICE_STATUS]['position']['status'] == 'OK'
+        return self._state.attributes[SERVICE_STATUS]['position']['status'] != 'DRIVER_DISABLED'
```

Comment:
```suggestion
        return self._state.attributes[SERVICE_STATUS]['position']['status'] not in ['DRIVER_DISABLED', 'TOO_FAR_AWAY']
```
Please also include `TOO_FAR_AWAY` for older vehicles (if the location of the smartphone/bimmer_connected is >2km from the vehicle).